testing/integration_test.sh

```diff
@@ -13,28 +13,29 @@ set -eo pipefail
 # Display commands being run.
 set -x
 
-cd github/cloud-profiler-nodejs
+cd $(dirname $0)/..
```

Comment:
This change is needed (or at least helpful) because integration_test.sh is now called from other bash scripts. So it's helpful that the the working directory doesn't need to be a particular directory when this is called.